CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -511,8 +511,8 @@
         <note>Accessibility label when a movie's poster is not available</note>
       </trans-unit>
       <trans-unit id="Movies and TV shows will show one link for quick access. The others are available in the More menu." xml:space="preserve">
-        <source>Movies and TV shows will show one link for quick access. The others are available in the More menu.</source><target>Filme und TV-Sendungen zeigen eine Schnellverknüpfung an. Die anderen Einträge kannst du unter `Mehr` finden.</target>
-        <note/>
+        <source>Movies and TV shows will show one link for quick access. The others are available in the More menu.</source><target>Filme und TV-Sendungen zeigen eine Schnellverknüpfung an. Die anderen Einträge kannst du unter `Mehr` `…` finden.</target>
```

Comment:
Straightforward: the button label has the "..."

Reply:
Any reason this couldn’t be

```
`Mehr …`
```

Rather than

```
`Mehr` `…`
```

…?

Reply:
Same kerning consideration as above, just here even uglier, probably, as the "..." is a single character :D

Reply:
We could also non-monospace it and do "Mehr ...", with straight quotes.

Reply:
Leave it as the way you have it for now, please. I'll try to remember to switch English too.